### PR TITLE
Update read me

### LINKED ISSUE #108: Ensure all ReadMes are up to date
### Task
- [x] Update archive readme
- [x] Update dashboard readme
- [x] Update database readme
- [x] Update national_rail readme
- [x] Update pdf_report readme
- [x] Update realtime_trains readme
- [x] Update terraform readme

### User Story
As a developer, I want to ensure all documentation is kept up to date so that if I ever come back to the project I understand what files are for what functionality.

### PR DESCRIPTION
**Pull request by Zhi-704**
Updated all readmes in relevant folders, removed test file that was no longer necessary as we have our own tests to pass status checks for. Close #108  and #104 and #41

### CODE REVIEW

File: national_rail/README.md [hunk 1]
```diff
@@ -32,11 +33,11 @@ The main script to interact with the National Rail API is `national_rail.py`. Yo
 python3 national_rail.py
 ```
 
-This will retrieve information about any incidents occurring in NationalRail. 
+This will retrieve information about any incidents occurring in NationalRail as well as alert any users of recent incidents.
```
Comment (FarihaChoudhury) [inline national_rail/README.md]: alert any subscribed users

File: README.md
Comment (A-Faris) [inline README.md:1]: Looks amazing, very clear

File: pdf_report/README.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # PDF Report
 
+<<<<<<< HEAD
+## Configuration
+Before running the script, you need to set up your National Rail API credentials. Create a new file called `.env` in the `pdf_report` directory and add the following lines:
+```text
+AWS_ACCESS_KEY=your_aws_access_key
+AWS_SECRET_KEY=your_aws_secret_access_key
+=======
```
Comment (FarihaChoudhury) [inline pdf_report/README.md]: merge conflicts stuff remains

File: national_rail/README.md
Comment (A-Faris) [inline national_rail/README.md:1]: Nice update

File: pdf_report/README.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 # PDF Report
 
+<<<<<<< HEAD
```
Comment (A-Faris) [inline pdf_report/README.md]: What's this for?

File: pdf_report/README.md
Comment (A-Faris) [inline pdf_report/README.md:1]: I think you had conflicts here and you may not have fixed them properly. Please remove the irrelevant parts

File: README.md
Comment (fm1psy) [reply README.md:1]: I agree, I like the level of detail in this readme

File: realtime_trains/README.md
Comment (A-Faris) [inline realtime_trains/README.md:1]: Nice update and fixing typos

**Review by A-Faris — DISMISSED** [review]
Looks great but the `pdf_report/README.md` file needs fixing

**Review by A-Faris — APPROVED** [review]
Looks good

**Pull request merged**